pkgs.nix

```diff
@@ -1,5 +1,13 @@
-import (
-  # let rev = "60cce7e5e1fdf62421ef6d4184ee399b46209366"; in
-  let rev = "04a48a2fcd2370f4e8a8c9fd3394531ccdb30f1e"; in
-  fetchTarball "https://github.com/MatrixAI/nixpkgs/archive/${rev}.tar.gz"
-)
+#import (
```

Comment:
Remove these comments and just write a comment on this PR about how you came to using the relevant commits.